src/@types/event.ts

```diff
@@ -120,6 +120,8 @@ export enum RoomType {
     ElementVideo = "io.element.video",
 }
 
+export const ToDeviceMessageId = "org.matrix.msgid";
```

Comment:
Is this specced anywhere to be using `org.matrix`? Doth it need be?

Reply:
Well, no it's not, and I hope not. I'm trying to keep it out of the spec as a quick thing implementations can add if they so choose (but nobody should rely on it in any way).

Reply:
I was just wondering if `org.matrix` had any special sentiment in the spec, given we do spec & use it - `org.matrix.custom.html` for example

Reply:
No, `org.matrix` is not special in the spec.

`org.matrix.custom.html` was *supposed* to be an "unstable development feature" and get turned into something else before speccing, but by the time that was on the cards it was way too late. It is literally the poster child for why I get cranky about using unstable prefixes in production.